app/models/case_transition.rb

```diff
@@ -24,9 +24,16 @@ class CaseTransition < ActiveRecord::Base
              class_name: 'Case::Base',
              foreign_key: :case_id
 
+  # This list should be bigger, but don't have time or inclination to move all event names here (yet)
```

Comment:
😂

I was only joking about a comment!

Ok as a compromise can you move these constants out of this file into somewhere in `initialisers` or something so we can consider using them across the site and possibly even in the state_machin/*.yml files!

Reply:
I thought the CaseTransition class seemed a reasonable place for these constants as they are all to do with case transitions...